pages/ourstory.vue

```diff
@@ -35,7 +35,15 @@
       height="810"
     />
 
-    <paragraphs path="ourstory.paragraphs" :from="3" :to="4" />
+    <paragraphs path="ourstory.paragraphs" :from="3" :to="3" />
+
+    <paragraph :margin-bottom="false">
+      <i18n path="ourstory.tripandpick.content" tag="p" class="paragraph-sm xl:paragraph">
+        <template #tripandpick>
+          <a class="text-link" :href="$t('ourstory.tripandpick.map.tripandpick.link')" target="_blank" rel="noopener noreferrer">{{ $t('ourstory.tripandpick.map.tripandpick.value') }}</a>
+        </template>
+      </i18n>
+    </paragraph>
```

Comment:
This will cause a hydration failure as reflected in the `console` warning message:
<img width="1912" alt="console_hydration_warning" src="https://user-images.githubusercontent.com/1261457/112123592-0f394c00-8bca-11eb-8b91-9b559a24b1da.png">

It's same problem I introduced when implementing payments(see the comments in #54 for details) and that caused the regression fixed in 
ed96bcc6232b7df602728a64289eceed893ff7c3.

What happens is that by specifying `tag="p"` for the `i18n` wrapper you end up with a DOM like this
<img width="967" alt="pp" src="https://user-images.githubusercontent.com/1261457/112124137-b1593400-8bca-11eb-9fbc-f5a948d40442.png">
Which is incorrect as you can't have a block level element `p` inside another block level element `p` and this causes the render to be different between client and server.  

In `dev` mode this falls back to client side render and you don't notice a problem.  

The problem only appears after doing `yarn generate` and serving the static website.
In this case I could not notice anything but we can't guarantee it will work correctly after accessing the page.

To fix this we simply need to remove the `tag="p"` overwrite as the default is `span` and the DOM will look like this
<img width="961" alt="nopp_but_class_duplicates" src="https://user-images.githubusercontent.com/1261457/112127206-da2ef880-8bcd-11eb-84a6-27119766920a.png">

The warning is gone but as you notice we have a duplicate `paragraph-sm xl:paragraph` class on `p` and child `span`, so the solution works but is not ideal.

As `<paragraph :margin-bottom="false"` is equivalent to `paragraph-sm xl:paragraph` we can just do
```suggestion
    <paragraphs path="ourstory.paragraphs" :from="3" :to="3" />

    <i18n path="ourstory.tripandpick.content" tag="p" class="paragraph-sm xl:paragraph">
      <template #tripandpick>
        <a class="text-link" :href="$t('ourstory.tripandpick.map.tripandpick.link')" target="_blank" rel="noopener noreferrer">{{ $t('ourstory.tripandpick.map.tripandpick.value') }}</a>
      </template>
    </i18n>
```
which results in a correct and clean DOM:

<img width="948" alt="equivalent_margin_bottom_false" src="https://user-images.githubusercontent.com/1261457/112128960-99d07a00-8bcf-11eb-9ddd-30b41f919628.png">

Reply:
Got it, changed it as suggested.